src/main/java/step3/utils/ValidationUtils.java

```diff
@@ -22,9 +28,23 @@ public static boolean validWinningNumbersComma(String winningNumbers) {
     }
 
     public static boolean validWinningNumbersDigit(int size) {
-        if (size != 6) {
+        if (size != LOTTO_END_DIGIT) {
             return false;
         }
         return true;
     }
+
+    public static Set<Integer> validWinningNumbers(String winningNumbers) {
+        if (!ValidationUtils.isNullCheck(winningNumbers)) {
+            throw new RuntimeException(WINNING_NUMBER_NULL_EXCEPTION);
+        }
+        if (!ValidationUtils.validWinningNumbersComma(winningNumbers)) {
+            throw new RuntimeException(WINNING_NUMBER_COMMA_EXCEPTION);
+        }
+        Set<Integer> winningNumberList = StringUtils.processStringNumbers(winningNumbers);
+        if (!ValidationUtils.validWinningNumbersDigit(winningNumberList.size())) {
+            throw new RuntimeException(WINNING_NUMBER_SIX_DIGIT_EXCEPTION);
+        }
+        return winningNumberList;
+    }
```

Comment:
메소드로 분리 잘해주셨네요 👍 

현재 `validWinningNumbers()` 메소드를 사용하는 쪽에서 생각해보면 `boolean 이나 void 리턴`을 기대할 것 같아요.
분리를 해보면 좋을 것 같은데 소정님 생각은 어떠신가요?

```suggestion
    public static Set<Integer> validWinningNumbers(String winningNumbers) {
        if (!ValidationUtils.isNullCheck(winningNumbers)) {
            throw new RuntimeException(WINNING_NUMBER_NULL_EXCEPTION);
        }
        
        if (!ValidationUtils.validWinningNumbersComma(winningNumbers)) {
            throw new RuntimeException(WINNING_NUMBER_COMMA_EXCEPTION);
        }
        
        Set<Integer> winningNumberList = StringUtils.processStringNumbers(winningNumbers);
        
        if (!ValidationUtils.validWinningNumbersDigit(winningNumberList.size())) {
            throw new RuntimeException(WINNING_NUMBER_SIX_DIGIT_EXCEPTION);
        }
        
        return winningNumberList;
    }
```